examples/tests/design.test.ts

```diff
@@ -0,0 +1,47 @@
+// design test for material.io
+
+import { shortest } from "@antiwork/shortest";
+
+// Material Design 3 color system
+shortest("Visit material.io and verify the Material Design 3 color system implementation. Primary color should be #6750A4 (purple) with tonal palette ranging from 0-100. Check dynamic color theming in light and dark modes. Verify key color roles: primary, secondary (#9C27B0), tertiary (#FFB4AB), and error (#B3261E) across components", {
+  baseColors: {
+    primary: "#6750A4",
+    secondary: "#9C27B0",
+    tertiary: "#FFB4AB",
+    error: "#B3261E"
+  },
+  modes: ["light", "dark"]
+});
```

Comment:
Long context description to help AI make more deterministic decisions when it comes to validating design systems. Passing color system as json format to help with readability and AI context.